src/navigator/TabNavigator.tsx

```diff
@@ -59,7 +59,7 @@ export default function TabNavigator({ route }: Props) {
           freezeOnBlur: false,
           lazy: false,
           tabBarLabel: t('bottomTabsNavigator.home.tabName') as string,
-          tabBarIcon: Logo,
+          tabBarIcon: ({ color }) => <Home color={color} size={20} />,
```

Comment:
It seems like `size` prop from `Tab.Screen` is always provided so in order to overwrite it but keep the ability to use that icon with different sizes - made it a callback.